Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: build
+ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+
+build:
+	python3 setup.py sdist
+	# Build bdist_wheel for local MacOS development - turn off OPENMP on Mac (but leave it alone for sdist)
+	OPENMP_DISABLED=true python3 setup.py sdist bdist_wheel
```

Comment:
Why do we need to run `setup.py sdist` twice?

Reply:
Oops good catch, fixing.